crawl-ref/source/dat/des/branches/swamp.des

```diff
@@ -2856,6 +3050,63 @@ Ztt..Kttttt..ttZ
 ZtrrrrtZZtqqqqtZ
 ENDMAP
 
+NAME:    nicolae_swamp_thats_no_hydra
+TAGS:    transparent no_pool_fixup
+DEPTH:   Swamp
+KMONS:   1 = kraken
+KFEAT:   @ = . / W
+KFEAT:   1 = w
+SHUFFLE: LMN, OPQ, RSZ
+SUBST:   LOR = @, MNPQSZ = t, - = .W, ` = wW, D = tww
+MAP
+     LLLLM
+   ZZLLLLMMM
+  ZZZ-----MMM
+ ZZ--`````--MM
+ ZZ-`DDDDD`-NN
+SS-`DDDDDDD`-NN
+SS-`DDwwwDD`-NN
+SS-`DDw1wDD`-NN
+SS-`DDwwwDD`-OO
+RR-`DDDDDDD`-OO
+ RR-`DDDDD`-OO
+ RR--`````--OO
+  RRQ-----PPP
+   RQQQQPPPP
+     QQQPP
+ENDMAP
+
+NAME:    nicolae_swamp_vermiculated
+TAGS:    transparent patrolling
+DEPTH:   Swamp
+KMONS:   ~ = steelbarb worm / swamp worm / tyrant leech
```

Comment:
Maybe have steelbarb at half weight so it stays more in branch theme? The center of the vault seems quite dense but it's ok since it also provides a ton of corners for the player to abuse.